.dependabot/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
+    automerged_updates:
+      - match:
+          dependency_type: "development"
+          update_type: "all"
+      - match:
+          dependency_type: "production"
+          update_type: "all"
+    commit_message:
+      prefix: "fix"
+      prefix_development: "chore"
+      include_scope: true
```

Comment:
There is a new config version (https://dependabot.com/docs/config-file-beta/) that I want to give a try with a config like this:

```yml
version: 2
updates:
  # create PRs for out-of-range updates
  - package-ecosystem: "npm"
    directory: "/"
    schedule:
      interval: "daily"
    versioning-strategy: "increase-if-necessary"
```

It will only send pull requests for out-of-range updates if I understand correctly, that will drastically reduce the noise

Looks like live updates will be deprecated though, everything will be on schedule 🤷

Reply:
interesting. i hadn't seen anything about v2 yet. 

> It will only send pull requests for out-of-range updates if I understand correctly, that will drastically reduce the noise

honestly, i'm not personally too concerned about noise as long as they are automerged. i normally define `save-exact=true` in an `.npmrc` for applications (not packages), so out-of-range isn't really an issue in that case. i'm a little surprised i havent configured that here, but probably comes down to thinking that i already had, haha.

i never pushed getting greenkeeper-keeper wired up to this project, but was hoping auto-merge built into dependabot would help in this case. sad to see that it is also on the deprecation list.

Reply:
good point, settings is an app, so we actually do want in-range updates anyway.

> as long as they are automerged

Looks like that is being deprecated with v2 ...

<img width="564" alt="image" src="https://user-images.githubusercontent.com/39992/81965127-7a8ab880-95cc-11ea-804b-60185c5d67d9.png">

Reply:
yeah, i saw that :(

not as big of an issue with this app since there are so few dependencies, but will certainly influence my usage elsewhere